src/EventSubscriber/FinishResponseSubscriber.php

```diff
@@ -1,26 +1,31 @@
 <?php
-/**
- * Event Subscriber.
- *
- */
-
-use Drupal\Core\EventSubscriber\FinishResponseSubscriber;
-use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
 
 namespace Drupal\nobots\EventSubscriber;
 
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
 /**
  * Response subscriber to handle finished responses.
  */
-class FinishResponseSubscriber extends \Drupal\Core\EventSubscriber\FinishResponseSubscriber {
+class FinishResponseSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    $events[KernelEvents::RESPONSE][] = ['onRespond'];
+    return $events;
+  }
 
   /**
    * Sets extra headers on successful responses.
    *
    * @param \Symfony\Component\HttpKernel\Event\FilterResponseEvent $event
    *   The event to process.
    */
-  public function onRespond(\Symfony\Component\HttpKernel\Event\FilterResponseEvent $event) {
+  public function onRespond(FilterResponseEvent $event) {
     $request = $event->getRequest();
```

Comment:
Probably can remove this line as well as it is not used.

Reply:
Just saw that. Ignore.

Reply:
Dang. Conflicts still came up.